whisper/cli/src/main.rs

```diff
@@ -16,16 +16,29 @@
 
 //! Whisper command line interface
 //!
+//! ## Overview
+//!
 //! Spawns an Ethereum network instance and attaches the Whisper protocol RPCs to it.
 //!
+//! ## Usage
+//!
+//! Whisper is not distributed with regular Parity Ethereum releases
```

Comment:
You sure about this? We do build it as part of the CI pipeline for sure.

Reply:
It is not true, https://vanity-service.parity.io/parity-binaries?format=markdown&version=v2.5.4

Reply:
i've pushed a change in this commit showing both approaches bc1a3e62dd27724c44d1d69d701ed9848b9520a6